src/main/java/org/fundacionjala/coding/Ana/Ean.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.coding.Ana;
+
+/**
+ * goods have an International Article Number.
+ *
+ * @author Ana Maria Mamani Zenteno.
+ */
+public class Ean {
+    private static final int ONE = 1;
+    private static final int THREE = 3;
+    private static final int DIVIDABLE = 10;
+//    private static final int TWO = 2;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)